client/gutenberg/extensions/tiled-gallery/edit.jsx

```diff
@@ -178,7 +179,7 @@ class TiledGalleryEdit extends Component {
 				<Fragment>
 					{ controls }
 					<MediaPlaceholder
-						icon="format-gallery"
+						icon={ <div className="tiled-gallery__media-placeholder-icon">{ icon }</div> }
```

Comment:
Bummer that `MediaPlaceholder` expects icon with `<svg className="dashicon" width="20" height="20">`...

We could either add those attributes to our icon SVG directly but it's probably safer to add styles via our own class selector since Core has been removing Dashicons in favour of Material icons.

Reply:
That is a bummer. Seems like something that should be changed sooner than later in core since core doesn't even use dashicons in Gberg.